test/global-setup.js

```diff
@@ -2,8 +2,6 @@ let grpcServer = require('../grpc/server')
 let queue = require('../queue/queue')
 
 let grpcClient = require('./grpc/client')
-let grpcMocks = require('./util/grpc-mocks')
```

Comment:
you can remove 'grpc-mock' dependency from `package.json` too - not used anymore